.github/workflows/update-bpmn-visualization-version.yml

```diff
@@ -15,33 +15,33 @@ jobs:
       VERSION: ${{ github.event.client_payload.version || github.event.inputs.version }}
     steps:
       - uses: actions/checkout@v3
-      - name: Get old bpmn-visualization version
+      - name: Get the old version of the bpmn-visualization TypeScript library
         id: lookupOldBPMNVisuVersion
         uses: mikefarah/yq@v4.30.5
         with:
-          cmd: yq '.dependencies[] | select(.name == "bpmn-visualization") | .version' inst/htmlwidgets/bpmnVisualization.yaml
+          cmd: yq '.dependencies[] | select(.name == "bpmn-visualization") | .version' inst/htmlwidgets/bpmnVisualizationR.yaml
       - run: echo "OLD_VERSION=${{ steps.lookupOldBPMNVisuVersion.outputs.result }}" >> $GITHUB_ENV
-      - name: Delete old bpmn-visualization js lib file
+      - name: Delete the old js file of the bpmn-visualization TypeScript library
```

Comment:
Too verbose, I suggest to restore the previous wording or at minimum, to remove "TypeScript library" everywhere.
I won't put a comment on all lines of the workflows, but this comment applies everywhere.

Reply:
```suggestion
      - name: Delete the old file of the bpmn-visualization
```